src/cascadia/TerminalApp/SnippetsPaneContent.xaml

```diff
@@ -0,0 +1,262 @@
+                    </Grid>
+                </mux:TreeViewItem>
+            </DataTemplate>
+
+            <ResourceDictionary.ThemeDictionaries>
+                <!--  same as in MainPage, this is SolidBackgroundFillColorTertiary  -->
+                <ResourceDictionary x:Key="Dark">
+                    <Color x:Key="PageBackground">#282828</Color>
+                    <Color x:Key="PlayButtonHoveredColor">#90ef90</Color>
+                    <Color x:Key="PlayButtonNormalColor">#8888</Color>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <Color x:Key="PageBackground">#F9F9F9</Color>
+                    <Color x:Key="PlayButtonHoveredColor">#257f01</Color>
+                    <Color x:Key="PlayButtonNormalColor">#88222222</Color>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <!--  Define resources for HighContrast mode here  -->
+                    <StaticResource x:Key="PageBackground"
+                                    ResourceKey="SystemColorWindowColorBrush" />
+                </ResourceDictionary>
+
+            </ResourceDictionary.ThemeDictionaries>
+
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <!--  Don't allow the user to focus the root pane, but do allow individual controls to be focused  -->
+
+    <Grid AllowFocusOnInteraction="False"
+          Background="{ThemeResource PageBackground}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <TextBlock x:Name="_title"
+                   x:Uid="SnippetPaneTitle"
+                   Grid.Row="0"
+                   Margin="4"
+                   FontSize="24" />
+
+        <TextBlock Grid.Row="1"
+                   Margin="8,16,8,8"
+                   FontSize="18"
+                   TextWrapping="WrapWholeWords"
+                   Visibility="{x:Bind mtu:Converters.InvertedBooleanToVisibility(HasSnippets), Mode=OneWay}">
+            <Run x:Uid="SnippetsPaneNoneFoundHeader" />
+            <LineBreak />
+            <Run x:Uid="SnippetsPaneNoneFoundDetails" />
+        </TextBlock>
+
+        <TextBox x:Name="_filterBox"
+                 x:Uid="SnippetsFilterBox"
+                 Grid.Row="1"
+                 Margin="8,0,8,8"
+                 AllowFocusOnInteraction="True"
+                 TextChanged="_filterTextChanged"
+                 Visibility="{x:Bind HasSnippets, Mode=OneWay}" />
+
+        <mux:TreeView x:Name="_treeView"
+                      Grid.Row="2"
+                      AllowFocusOnInteraction="True"
+                      CanDragItems="False"
+                      CanReorderItems="False"
+                      ItemTemplate="{StaticResource TaskItemTemplate}"

```

Comment:
```suggestion
                      ItemTemplate="{StaticResource TaskItemTemplate}"
                      SelectionMode="None"
```

Reply:
I'm hoping this gets rid of the pill on the left. We don't need it (and I found a case or two where it doesn't move with focus).

Reply:
This actually makes it impossible to do the "click to see the whole input string" thing. So this I'm gonna reject